src/pages/WriteProductsPage.tsx

```diff
@@ -1,9 +1,67 @@
 import type React from 'react';
+import { useForm, SubmitHandler } from 'react-hook-form';
+
+type FormData = {
+	title: string;
+	url: string;
+	comment: string;
```

Comment:
```suggestion
	comment?: string;
```
ここも必須入力じゃないので、こんな感じで。